inc/command/class-command.php

```diff
@@ -212,19 +212,33 @@ protected function phpunit( InputInterface $input, OutputInterface $output ) {
 	 */
 	protected function codecept( InputInterface $input, OutputInterface $output ) {
 		$options = $input->getArgument( 'options' );
-		$module = $input->getOption( 'module' );
+		$tests_folder = $input->getOption( 'path' );
 		$run_headless_browser = $input->getOption( 'browser' );
 		$use_chassis = $input->getOption( 'chassis' );
-		$tests_folder = $module !== 'project' ? "altis/$module/tests" : '../tests';
 		$project_subdomain = $this->get_project_subdomain();
 
+		$folders = [
+			'_data' => '',
+			'_support' => '',
+			'_envs' => '',
+		];
+
+		foreach ( $folders as $folder => $_ ) {
+			if ( file_exists( $tests_folder . '/' . $folder ) ) {
+				$folders[ $folder ] = $tests_folder . '/' . $folder;
+			} else {
+				$folders[ $folder ] = 'altis/dev-tools/tests/' . $folder;
+			}
+		}
+
 		// Write the default config.
 		$config = [
 			'paths' => [
 				'tests' => $tests_folder,
 				'output' => '../tests/_output',
```

Comment:
My original comments about outputs being somewhere visible were based on using this command for running project level tests - I'm wondering if we specify a path we should just use the `_output` directory relative to the provided path? It'll still default to `../tests/_output`. What are your thoughts there?

Reply:
I think a single place would be best, since you can run multiple test commands with different folders, and it might help if they all drop artifacts to the same directory for the sake of convenience.

Reply:
Yeah, just thinking what someone's expectations are most likely to be and whether this fits with that, even if it might be more convenient for us internally.

A project might not have tests in a `tests` directory in the project root but in a plugin / theme / module for example, but I'd then have to be looking in the project root `tests` directory for the outputs even though there may not be any tests there.

Outputs being relative to the tests directory being run seems to me like the most obvious and easiest set up to understand. There's also potential for collisions on test names (however unlikely) if running multiple test commands with different folders. In those circumstances I believe the outputs directory could be overridden with the `-o` option if we did want to a single location.

Reply:
Yeah, `-o` to consolidate while defaulting to output being next to tests does make sense! best of both worlds.